source/uwp/Renderer/lib/TileControl.cpp

```diff
@@ -196,9 +196,9 @@ namespace AdaptiveNamespace
         // the container until we get the right dimensions.
         if (m_imageSize.Width != 0 && m_imageSize.Height != 0)
         {
-            switch (mode)
```

Comment:
no case for "Cover"?

Reply:
So, this is just a rename, and I'm not 100% familiar with the code here, but, i believe this is the code that handles tiling the images so cover (previously known as stretch) isn't a relevant case.  

---
In reply to: [285204074](https://github.com/microsoft/AdaptiveCards/pull/2928#discussion_r285204074) [](ancestors = 285204074)